### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by seemiller**
## What this PR does / why we need it

Add owners teams for secretgen-controller and whereabouts packages

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -68,6 +68,9 @@ addons/packages/pinniped @vmware-tanzu/pkg-pinniped-owners
 ## Ownership of the prometheus package
 addons/packages/prometheus @vmware-tanzu/pkg-prometheus-owners
 
+## Ownership of the secretgen-controller package
+addons/packages/prometheus @vmware-tanzu/pkg-secretgen-controller-owners
```
Comment (stmcginnis) [inline CODEOWNERS:72]: Missed updating the path?

File: CODEOWNERS [hunk 2]
```diff
@@ -76,3 +79,6 @@ addons/packages/vsphere-cpi @vmware-tanzu/pkg-vsphere-cpi-owners
 
 ## Ownership of the vsphere-csi package
 addons/packages/vsphere-csi @vmware-tanzu/pkg-vsphere-csi-owners
+
+## Ownership of the whereabouts package
+addons/packages/vsphere-csi @vmware-tanzu/pkg-whereabouts-owners
```
Comment (stmcginnis) [inline CODEOWNERS:84]: Same

File: CODEOWNERS (hunk 1 above)
Comment (seemiller) [reply CODEOWNERS:72]: d'Oh!

File: CODEOWNERS (hunk 2 above)
Comment (seemiller) [reply CODEOWNERS:84]: d'Oh!

**Review by stmcginnis — APPROVED** [review]
Looks good!

**Pull request merged**